VK_App/VK_App/ViewControllers/SecondBarVC/SecondBarVC/UnFollowTableViewCell.swift

```diff
@@ -20,8 +24,23 @@ final class UnFollowTableViewCell: UITableViewCell {
 
     // MARK: Public Methods
 
-    func unFollowGroupInfo(_ group: Group) {
-        unFollowGroupLabel.text = group.groupName
-        unFollowImageView.image = UIImage(named: group.groupImage)
+    func configure(group: Group) {
+        unFollowImageView.image = nil
+        currentPath = group.photo50Path
+        unFollowGroupLabel.text = group.name
+        getImage(imagePath: group.photo50Path)
+    }
+
+    // MARK: Private Methods
+
+    private func getImage(imagePath: String) {
+        LoadingImage.shared.getImage(imagePosterPath: imagePath) { [weak self] data in
+            DispatchQueue.main.async {
+                guard let self = self else { return }
+                if imagePath == self.currentPath {
+                    self.unFollowImageView.image = UIImage(data: data)
+                }
+            }
+        }
```

Comment:
```suggestion
    private func getImage(imagePath: String) {
        LoadingImage.shared.getImage(imagePosterPath: imagePath) { [weak self] data in
            DispatchQueue.main.async {
                guard let self = self else { return }
                if imagePath == self.currentPath {
                    self.unFollowImageView.image = UIImage(data: data)
                }
            }
        }
```
я б посоветовала этот код в extension вынести раз он у тебя повторяется в каждой ячейке, справишься?